src/prefect/orion/models/work_queues.py

```diff
@@ -57,13 +58,16 @@ async def create_work_queue(
             data["work_pool_id"] = default_agent_work_pool.id
 
     if data.get("work_pool_id"):
```

Comment:
Sorry (and this doesn't need to change here) but this will _always_ be true since you set a value for this above?

Reply:
Yup! Took it out